qa/tasks/qemu.py

```diff
@@ -354,9 +354,13 @@ def run_qemu(ctx, config):
                 ]
             )
 
+        nfs_service_name = 'nfs'
+        if float(remote.os.version) >= 8:
```

Comment:
Shouldn't we also test for `remote.os.type`?

Reply:
thought it seems to work on `ubuntu 18.04` according to your teuthology runs?

Reply:
There is another override within those functions based on the packaging type.

Reply:
I added an explicit callout for rhel and centos here for clarity.